microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -929,7 +929,7 @@ You can also choose to have Microsoft Edge auto-group all of your tabs, by using
 
 All tab information that's sent to Microsoft is deleted immediately after processing occurs. No data is stored for any period of time. 
 
-The Microsoft Edge Tab organization behavior is turned on by default. To turn this behavior off, go to `edge://settings/privacy` and under **Services** turn off the **Let Microsoft Edge help keep your tabs organized** setting:
+The Microsoft Edge Tab organization behavior is turned on by default. To turn this behavior off, go to `edge://settings/privacy`<!--todo: check, maybe replace by GUI nav path--> and under **Services** turn off the **Let Microsoft Edge help keep your tabs organized** setting:
```

Comment:
this should be: **edge://settings/searchServices**

 **Settings and more** (**...**) > **Settings** > **Search and services**.

Reply:
```
The Microsoft Edge Tab organization behavior is turned on by default. 
To turn this behavior off, go to **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) > **Settings** > **Search and services**, 
and then turn off the **Let Microsoft Edge help keep your tabs organized** setting.

When the Tab Organization service is turned off, new Tab groups are given a generic name 
such as **Group 1**, and you can then rename the Tab group to be more descriptive.  
```
deleted outdated png shown at https://learn.microsoft.com/en-us/microsoft-edge/privacy-whitepaper/#tab-organization